docs/spec/transactions/transaction.json

```diff
@@ -55,6 +55,10 @@
                 }
             },
             "additionalProperties": false
+        },
+        "sampled": {
+            "type": "boolean",
```

Comment:
`"type": ["boolean", "null"]` otherwise you cannot send the key with an empty value.